src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -482,7 +482,9 @@ public static void MapInputTransparent(IViewHandler handler, IView view)
 #if ANDROID
 			handler.UpdateValue(nameof(IViewHandler.ContainerView));
 
-			if (handler.ContainerView is WrapperView wrapper)
+			if (handler.PlatformView is IInputTransparentManagingView managing)
+				managing.InputTransparent = view.InputTransparent;
```

Comment:
If the native view supports managing its own transparency state, then use that instead of blindly wrapping.